src/Microsoft.ML.OnnxTransformer/Microsoft.ML.OnnxTransformer.csproj

```diff
@@ -9,7 +9,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
-    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="$(MicrosoftMLOnnxRuntimePackageVersion)" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime.Gpu" Version="$(MicrosoftMLOnnxRuntimePackageVersion)" />
```

Comment:
We can't change this. Or else people without GPUs won't be able to use the OnnxTransform. See [https://github.com/microsoft/onnxruntime/issues/2184](https://github.com/microsoft/onnxruntime/issues/2184)